test/jdk/java/security/KeyStore/PKCS12/CheckNullDefault.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Where did you get the copyright from? IIRC, there should be  a coma after the year and also copyright for the test file should not have the "Classpath" thing.

Reply:
You can check around tests under the same directory for references. There is a recent fix which sweep through copyright typos and `CheckDefaults` happen to be one of them. You had the old one here and should be updated. Besides there is always a coma after the year convention-wise.